src/Models/ProjectRepository.coffee

```diff
@@ -33,6 +33,12 @@ class ProjectRepository extends BaseModel
     @debug "Projects::listTags()"
     @get "projects/#{Utils.parseProjectId projectId}/repository/tags", (data) => fn data if fn
 
+  # === Compare
+  compare: (projectId, from, to, fn = null) =>
+    @debug "Projects::compare()"
+    params = from: from, to: to
+    @get "projects/#{Utils.parseProjectId projectId}/repository/compare/", params, (data) => fn data if fn
```

Comment:
I just write code as other methods in this file. I assume `compare` is not the only method with unnecessary fat arrow and to long line length.

Personnaly, I can close this PR because I do not use Gitlab anymore. @fbourigault, are you interested by this PR?